Cargo.toml

```diff
@@ -66,24 +67,29 @@ clap = { version = "4.5", optional = true, features = ["derive"] }
 num = { version = "0.4", default-features = false, features = ["alloc", "libm"] }
 num-complex = { version = "0.4", default-features = false }
 rand = { version = "0.8", default-features = false }
+rand_chacha = { version = "0.3", default-features = false }
```

Comment:
Are we now using this in non-test code?

Reply:
It is used in the `prove` method to generate a seed for the PRNG for zk. This will be removed once we find a solution to do deterministic signature generation.